packages/data-point/lib/entity-types/entity-request/__snapshots__/resolve.test.js.snap

```diff
@@ -0,0 +1,24 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`resolveRequest log errors when request fails 1`] = `
+"404 - \\"not found\\"
+
+Entity info:
+  - Id: test:test
+  - options: { json: true,
```

Comment:
im using node's util.inspect, the most I can format it is to break on first property. JSON.stringify would do this, but it does not handle circular references, and that is way worse than this format part..

Reply:
OK, that's a fair point. Thanks.